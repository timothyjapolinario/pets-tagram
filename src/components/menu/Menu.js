import homeIcon from "../../assets/home-icon.svg";
import searchIcon from "../../assets/search-icon.png";
import messengerIcon from "../../assets/messenger-icon.svg";
import heartIcon from "../../assets/heart-icon.png";
import plusIcon from "../../assets/plus-icon.png";
import profileImg from "../../assets/profile.jpg";
import menuIcon from "../../assets/menu-icon.png";
const Menu = () => {
  return (
    <div className="flex min-h-[80vh] flex-col gap-[6vh] pl-[1.3vw] pr-[7vw]">
      <h1 className="text-3xl font-light my-[4vh]">PetsTagram</h1>
      <div className="flex gap-4 items-center">
        <img src={homeIcon} className="h-[5vh]" />
        <span>Home</span>
      </div>
      <div className="flex gap-4 items-center">
        <img src={searchIcon} className="h-[5vh]" />
        <span>Search</span>
      </div>
      <div className="flex gap-4 items-center">
        <img src={messengerIcon} className="h-[5vh]" />
        <span>Messages</span>
      </div>
      <div className="flex gap-4 items-center">
        <img src={heartIcon} className="h-[5vh]" />
        <span>Notifications</span>
      </div>
      <div className="flex gap-4 items-center">
        <img src={plusIcon} className="h-[5vh]" />
        <span>Create</span>
      </div>
      <div className="flex gap-4 items-center">
        <img src={profileImg} className="h-[5vh] rounded-full" />
        <span>Profile</span>
      </div>
      <div className="flex gap-4 items-center my-5">
        <img src={menuIcon} className="h-[5vh]" />
        <span>Menu</span>
      </div>
    </div>
  );
};

export default Menu;
