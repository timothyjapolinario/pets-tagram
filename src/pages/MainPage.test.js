import React from "react";
import { render } from "@testing-library/react";
import Menu from "../components/menu/Menu";
import "@testing-library/jest-dom";
describe("Main Page Component", () => {
  it("renders correct menu", () => {
    const { getByRole, container } = render(<Menu />);
    expect(getByRole("heading").textContent).toMatch(/PetsTagram/i);
    expect(container).toMatchSnapshot();
  });
});
