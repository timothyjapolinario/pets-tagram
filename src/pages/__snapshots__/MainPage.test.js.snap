// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main Page Component renders correct menu 1`] = `
<div>
  <div
    class="flex min-h-[80vh] flex-col gap-[6vh] pl-[1.3vw] pr-[7vw]"
  >
    <h1
      class="text-3xl font-light my-[4vh]"
    >
      PetsTagram
    </h1>
    <div
      class="flex gap-4 items-center"
    >
      <img
        class="h-[5vh]"
        src="home-icon.svg"
      />
      <span>
        Home
      </span>
    </div>
    <div
      class="flex gap-4 items-center"
    >
      <img
        class="h-[5vh]"
        src="search-icon.png"
      />
      <span>
        Search
      </span>
    </div>
    <div
      class="flex gap-4 items-center"
    >
      <img
        class="h-[5vh]"
        src="messenger-icon.svg"
      />
      <span>
        Messages
      </span>
    </div>
    <div
      class="flex gap-4 items-center"
    >
      <img
        class="h-[5vh]"
        src="heart-icon.png"
      />
      <span>
        Notifications
      </span>
    </div>
    <div
      class="flex gap-4 items-center"
    >
      <img
        class="h-[5vh]"
        src="plus-icon.png"
      />
      <span>
        Create
      </span>
    </div>
    <div
      class="flex gap-4 items-center"
    >
      <img
        class="h-[5vh] rounded-full"
        src="profile.jpg"
      />
      <span>
        Profile
      </span>
    </div>
    <div
      class="flex gap-4 items-center my-5"
    >
      <img
        class="h-[5vh]"
        src="menu-icon.png"
      />
      <span>
        Menu
      </span>
    </div>
  </div>
</div>
`;
